crashsimilarity/downloader.py

```diff
@@ -124,7 +125,7 @@ def download_day_crashes(self, day, product='Firefox', offset=0, crashes_per_req
                 break
 
     @staticmethod
-    def download_and_save_crashes(days, product='Firefox', save_to_dir=utils.CRASHSIMILARITY_DATA_DIR):
+    def download_and_save_crashes(days, product='Firefox', save_to_dir=_CRASHSIMILARITY_DATA_DIR):
```

Comment:
Won't work (calling a class inside itself), what about changing the function to a class function instead of static? then modify the variable call?